doc/design/libsvm_format_column.md

```diff
@@ -0,0 +1,112 @@
+# Support LibSVM format on SQLFlow
+
+## Background
+
+SQLFlow extends SQL syntax to support End-to-End machine learning pipeline. 
+However, due to the variety of the data format (like CSV, LibSVM, etc.), 
+data pre-processing is usually a tough task for users. Users have to speed lots of time 
+to transform the raw inputs into the data format which the machine learning model accepts.
+SQLFlow should do some automatic data pre-processing to make End-to-End machine learning pipeline 
+easier.
+
+LibSVM is a commonly used data format in machine learning, and it is officially supported in many machine learning frameworks, 
+such as [XGBoost](https://xgboost.readthedocs.io/en/latest/tutorials/input_format.html) and 
+[SkLearn](https://scikit-learn.org/stable/modules/generated/sklearn.datasets.load_svmlight_file.html). There have been 
+[some issues](https://github.com/sql-machine-learning/sqlflow/issues/2323) that require SQLFlow should support LibSVM format column. 
+Therefore, SQLFlow should support to process LibSVM format to simplify the End-to-End machine learning pipeline.
+
+LibSVM format is more portable than CSV format when storing sparse data. Each line of the LibSVM file is like:
```

Comment:
LibSVM format => LibSVM data file format

Reply:
Added the "LibSVM format file" definition at the beginning of the design doc.